libopenage/renderer/camera/camera.h

```diff
@@ -15,6 +15,7 @@
 #include "renderer/camera/definitions.h"
 #include "renderer/camera/frustum_2d.h"
 #include "renderer/camera/frustum_3d.h"
+#include <optional>
```

Comment:
Small nitpick, but imports of standard libraries should be placed before local includes in our code. See our (arguably a bit hidden) code style examples:

https://github.com/SFTtech/openage/blob/66098cea4d799634d66b0e9b9691fdf715a7118d/doc/code_style/mom.cpp#L17-L32